Azure-Samples/azure-search-java-samples
# Java getting started sample

## Purpose

Update code to Java 11. Mostly this was change to use `HttpClient`. Some URLs changed (maybe original used old API?). 

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[X ] Code style update (formatting, local variables)
[X ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Edit src/main/resources/azure_search_config w. appropriate service name and API key
* Execute Maven goal `verify exec:java` 

## What to Check
Verify that the following are valid
* Should build, run tests (of "*Options" classes), and:
    * Create index (deleting existing if there) 
    * Perform indexing
    * Search and advanced search 
    * suggestion request

@HeidiSteen
